core/app/views/refinery/admin/_menu.html.erb

```diff
@@ -1,25 +1,33 @@
-<button type="button" class="btn btn-primary sidebar-left-opener fixed-t-l z-5 hidden-sm-up"><i class="icon icon-bars"></i></button>
+<nav>
```

Comment:
Maybe we can place this <nav> just inside the section#sidebar-left of just wrap the div#sidebar-left-inner.  It just seems odd to wrap 2 'position: fixed' elements in a <nav>.  As well as I've been going by a rule to have all 'dimmer' elements and 'sidebar' elements as direct children of the <body> element.  It looks like it still works in this case, especially since we decided not to make use of a dimmer, but I still think it makes sense to keep the sidebar a direct child of the <body> if possible.